src/platform/plugins/shared/metrics_experience/server/lib/fields/enrich_metric_fields.ts

```diff
@@ -112,8 +115,15 @@ export async function sampleMetricMetadata({
         size: 1,
         terminate_after: 1,
         query: {
-          exists: {
-            field,
+          bool: {
+            filter: [
+              {
+                exists: {
```

Comment:
We can use `existsQuery` [helper](https://github.com/elastic/kibana/blob/668aee1bb0c9027a444a8d1c898190c956b73059/x-pack/solutions/observability/plugins/observability/server/utils/queries.ts#L86-L89) like in many other queries

Reply:
That only exists in the X-Pack Observability Plugin, I wouldn't want to include that. This is such a simple query, I'm not sure why we need a helper.

Reply:
Ah, true, it's fine 👍